### PR TITLE
Additions to Bosch's Copyright statements

### PR DESCRIPTION
**Pull request by sschuberth**
Please have a look at the individual commit messages for the details.

### CODE REVIEW

**Comment by sschuberth** [comment]
FYI @mkurzman and @Sheriff-Manzoor.

File: .reuse/dep5 [hunk 1]
```diff
@@ -45,7 +45,7 @@ Copyright: 2017-2020 HERE Europe B.V. <opensource@here.com>
 License: Apache-2.0
 
 Files: clients/fossid-webapp/src/test/assets/*
-Copyright: 2020-2021 Bosch.IO GmbH
+Copyright: 2020-2021 Bosch.IO GmbH <osm@bosch.com>
```
Comment (Sheriff-Manzoor) [inline .reuse/dep5:48]: Why is OSM@bosch.com added to the copyrights statement? Curious to know if this was decided somewhere.

**Review by Sheriff-Manzoor — COMMENTED** [review]
Why is OSM@bosch.com added to the copyrights statement? Curious to know if this was decided somewhere.

File: .reuse/dep5 (hunk 1 above)
Comment (sschuberth) [reply .reuse/dep5:48]: I had pinged @mkurzman quickly in a chat to ask for which general email address to use here, and that's simply what we cam up with. So it's not a fixed decision, but a proposal. Any objections?

**Comment by Sheriff-Manzoor** [comment]
So this is already discussed. No objection from my end.

**Review by oheger-bosch — APPROVED** [review]

**Pull request merged**